ocaml/xapi/xapi_host.ml

```diff
@@ -3055,18 +3058,23 @@ let try_restart_device_models_for_recommended_guidances ~__context ~host =
       None
 
 let apply_recommended_guidances ~__context ~self =
+  (* This function runs on master host *)
```

Comment:
While the comment is helpful, it's better to enforce this property instead with code, for example an assertion that can raise an internal server error if the function is not run in the coordinator